src/app/flight-booking/models/flt-results-sort-comparators.ts

```diff
@@ -0,0 +1,41 @@
+import {Flight} from './flight';
+
+
+export function durationComparator(a: Flight, b: Flight): number {
+  const legsA = a.getLegs();
+  const legsB = b.getLegs();
+  const departureTimeA = legsA[0].departTime;
+  const arrivalTimeA = legsA.length === 2 ? legsA[1].arrivalTime : legsA.length === 1 ? legsA[0].arrivalTime : undefined;
+  const departureTimeB = legsB[0].departTime;
+  const arrivalTimeB = legsB.length === 2 ? legsB[1].arrivalTime : legsB.length === 1 ? legsB[0].arrivalTime : undefined;
+  const aDiff = arrivalTimeA.getTime() - departureTimeA.getTime();
+  const bDiff = arrivalTimeB.getTime() - departureTimeB.getTime();
```

Comment:
might cause issues since we are not accounting for time zones

Reply:
Yea the time zones are going to be on next sprint.